tests/run/utf16encoded.flags

```diff
@@ -0,0 +1 @@
+-encoding UTF16
```

Comment:
.flags file are currently ignored by partest, and I'm not sure if there's any way to have a run test with a special set of flags currently.
/cc @DarkDimius : If we make a `run-special` folder like `pos-special`, will its content be treated as run-tests by partest?